cypress/integration/onboardingTests.js

```diff
@@ -0,0 +1,184 @@
+// to launch test runner: yarn run cypress open
+
+describe("Onboarding", () => {
+  it("should login and logout", () => {
+    cy.visit("/");
+    cy.get("button")
+      .contains("Dev Bypass")
+      .click();
+    cy.url().should("include", "/redirect");
+    cy.wait(10000);
+    // check logged in
+    cy.url({ timeout: 30000 }).should("include", "/home");
+    cy.get("button")
+      .contains("Logout")
+      .click();
+    // check logged out
+    cy.url().should("eq", "http://localhost:3000/"); // => true
+  });
+  it("should add a class to year 1 summer 2", () => {
+    cy.visit("/");
+    cy.get("button")
+      .contains("Dev Bypass")
+      .click();
+    cy.url({
+      timeout: 30000,
+    }).should("include", "/home"); // wait up to 30 seconds to load the home screen
+    // click add class
+    cy.get(
+      ":nth-child(3) > .sc-kjoXOD > :nth-child(4) > .sc-fAjcbJ > .sc-caSCKo > .sc-hSdWYo > .MuiPaper-root > .sc-hzDkRC > .sc-jhAzac"
```

Comment:
these selectors will all be updated to use "data-cy" attributes which will make selecting elements more robust